### DISCUSSION TITLE
从2.26.x更新上来之后似乎无法上线

### DISCUSSION THREAD

**Discussion by Eudiplozoon** [Tsuk1ko/CQ-picfinder-robot#190] [问题求助]
今天更新到2.28.5之后bot好像无法正常上线，配置项里除了把新加的几行复制进去之外没动其他的
也试过用新的默认配置文件还是不能上线，求教是哪里出现问题

**Comment by Eudiplozoon** [comment]
![QQ截图20210521175131](https://user-images.githubusercontent.com/37258131/119120152-853d2080-ba5e-11eb-92af-896e770e3e1b.png)
![QQ截图20210521175148](https://user-images.githubusercontent.com/37258131/119120172-8a9a6b00-ba5e-11eb-8507-9e21e441bb07.png)

**Reply by Tsuk1ko** [reply]
提供一下 cqps 的日志

**Reply by Eudiplozoon** [reply]
{ /root/1/src/cache.js:1
Error: Cannot find module '@msgpack/msgpack'
Require stack:
- /root/1/src/cache.js
- /root/1/main.js
- /root/1/index.js
    at Object.<anonymous> (/root/1/src/cache.js:1)
  code: 'MODULE_NOT_FOUND',
  requireStack:
   [ '/root/1/src/cache.js', '/root/1/main.js', '/root/1/index.js' ] }

**Reply by Tsuk1ko** [reply]
没 npm i 或者 yarn 更新依赖

**Reply by Eudiplozoon** [reply]
啊 我之前用yarn是这个情况，刚才用npm i更新了一遍现在正常了
还有问一下setu结果合并转发那里是需要另外配置cqhttp的什么插件吗，默认开启之后群内搜图会无回复，私聊有回复

**Reply by Tsuk1ko** [reply]
你的 cqhttp 版本可能太旧了
